posthog/api/test/test_csp.py

```diff
@@ -275,8 +276,11 @@ def test_process_csp_report_from_query_params(self):
         assert event["properties"]["$session_id"] == "test-session"
         assert event["properties"]["$csp_version"] == "1"
 
-    def test_sample_csp_report(self):
-        # Create test properties
+    @patch("posthog.api.csp.datetime")
```

Comment:
nit: it's fine either way but we tend to use `freezegun` in tests to fix the datetime